dist/gaia-header-es5.js

```diff
@@ -658,10 +658,10 @@
            * Exports
            */
 
-          var base = window.GAIA_ICONS_BASE_URL || window.COMPONENTS_BASE_URL || "bower_components/";
+          var base = window.GAIA_ICONS_BASE_URL || window.COMPONENTS_BASE_URL || 'bower_components/';
 
           if (!document.documentElement) {
-            window.addEventListener("load", load);
+            window.addEventListener('load', load);
```

Comment:
all these changes are weird too; Are you sure you use the same version of bower ? configured the same way ?

I would lke the change of " to ', expecially relatively to gaia's coding style, but I wouldn't like that a future patch changes this back.